src/us/kbase/workspace/database/Workspace.java

```diff
@@ -120,7 +121,6 @@ public Workspace(
 		nonNull(listeners, "listeners");
 		noNulls(listeners, "null item in listeners");
 		this.listeners = Collections.unmodifiableList(listeners);
-		db.setResourceUsageConfiguration(rescfg);
```

Comment:
The DB layer no longer needs access to the resource config since the data size calcs happen in the business layer. There are several changes around this